spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/model/PetRepository.java

```diff
@@ -32,17 +27,18 @@
  * @author Michael Isvy
  * @author Maciej Szarlinski
  */
-public interface PetRepository extends JpaRepository<Pet, Integer> {
-
-    /**
-     * Retrieve all {@link PetType}s from the data store.
-     * @return a Collection of {@link PetType}s.
-     */
-    @Query("SELECT ptype FROM PetType ptype ORDER BY ptype.name")
-    List<PetType> findPetTypes();
-
-    @Query("FROM PetType ptype WHERE ptype.id = :typeId")
-    Optional<PetType> findPetTypeById(@Param("typeId") int typeId);
+@Repository
+public interface PetRepository extends CosmosRepository<Pet, Integer> {
+
+//    /**
```

Comment:
If these are not going to be done/used you should remove them

Reply:
Removed. Not needed as of now. 

> If these are not going to be done/used you should remove them